src/modules/CSADigitizer/README.md

```diff
@@ -0,0 +1,60 @@
+# CSADigitizer
+**Maintainer**: Annika Vauth (<annika.vauth@desy.de>)
+**Status**: Immature
+**Input**: PixelCharge  
+**Output**: PixelHit  
+
+### Description
+Digitization module which translates the collected charges into a digitized signal, emulating a charge sensitive amplifier with Krummenacher feedback.
+For this purpose, a transfer function for a CSA with Krummenacher feedback is taken from [@kleczek] : 
+$`H(s) = \frac{R_f}{((1+ \tau_f s) * (1 + \tau_r s))}, `$
+with $`\tau_f = R_f C_f `$ , rise time constant $`\tau_r = \frac{C_{det} * C_{out}}{g_m * C_f} `$ 
+
+The impulse response function of this transfer function is convoluted with the charge pulse.
+This module can be steered by either entering all contributions to the transfer function as parameters, or a simplified parametrisation in which the other parameters are calculated by the software (see e.g. [@binkley] for calculation of transconductance).
+
+Noise can be applied to the pulse, drawn from a normal distribution.
+For the amplified pulse signal, Time-of-Arrival and Time-over-Threshold can be counted.
+
+
+
+### Parameters
+* `model` :  Choice between different CSA models. Currently implemented are two parametrisations of the circuit from [@kleczek], SIMPLE and CSA.
```

Comment:
I would name them exactly as we use them in the code, i.e. lower-case. Also, if it's a parameter I think it helps to put it in backticks to be typeset as code: 

> Two parametrizations are available, `simple` and `csa` as explained in detail in the module description above.